platform/AppStoreConnect/Sources/AppStoreConnect/APIProvider+Sync.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+import AppStoreConnect_Swift_SDK
+
+/// Synchronous variants for AppStoreConnect-Swift-SDK API.
+extension APIProvider {
+    func request<T: Decodable>(_ endpoint: APIEndpoint<T>) async throws -> T {
```

Comment:
Removed semaphores, and used `async await`. Didn't know it was possible to use for Command Line Tools, neat.